torch/_inductor/codegen/cpp_wrapper_cpu.py

```diff
@@ -1912,6 +1918,25 @@ def codegen_conditional(self, conditional):
         self.writeline(ExitSubgraphLine(self))
         self.writeline("}")
 
+    def codegen_subgraph(self, subgraph, outer_inputs, outer_outputs):
```

Comment:
Wondering why we still need this method on the CPU wrapper subclass, if we have `codegen_subgraph_by_inlining` doing the same thing in the parent class? Is it because we want to use the prefix and suffix codegening functions defined here? If so, why do we need `codegen_subgaph_by_inlining` in the parent?

Reply:
Thanks for noticing. I can just fallback to `codegen_subgraph_by_inlining` here.